grace/io/core.py

```diff
@@ -194,3 +196,41 @@ def read_graph(filename: os.PathLike) -> GraceFileDataset:
     with GraceFile(filename) as gfile:
         data = gfile.read()
     return data
+
+
+def star_to_graph(filename: os.PathLike) -> nx.Graph:
+    """Reads a starfile into a graph."""
+    star_df = starfile.read(filename)
+    star_df = star_df.rename(
+        columns={"rlnCoordinateX": "x", "rlnCoordinateY": "y"}
+    )
+
+    return graph_from_dataframe(star_df)
+
+
+def mkdir_grace_from_star(
+    stardir: os.PathLike, gracedir: os.PathLike = None
+) -> None:
+    """Make and populate a grace directory from a directory of starfiles.
+
+    Inputs:
```

Comment:
Maybe change to numpy format so it's consistent with the rest of the repo? I.e., "parameters" instead of "inputs" etc